ModularTegustation/tegu_items/refinery/crates/misc.dm

```diff
@@ -6,25 +6,25 @@
 	rarechance = 15
 	cosmeticchance = 25
 	lootlist =	list(
-		/obj/item/ego_weapon/mini/hayong,
-		/obj/item/ego_weapon/shield/walpurgisnacht,
-		/obj/item/ego_weapon/lance/suenoimpossible,
-		/obj/item/ego_weapon/shield/sangria,
-		/obj/item/ego_weapon/mini/soleil,
-		/obj/item/ego_weapon/taixuhuanjing,
-		/obj/item/ego_weapon/revenge,
-		/obj/item/ego_weapon/shield/hearse,
-		/obj/item/ego_weapon/mini/hearse,
-		/obj/item/ego_weapon/raskolot,
-		/obj/item/ego_weapon/vogel,
-		/obj/item/ego_weapon/nobody,
-		/obj/item/ego_weapon/ungezifer,
-		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat,
-		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat_short,
+		/obj/item/ego_weapon/mini/city/hayong,
```

Comment:
This doesn't do anything. There is no mini/city path, what you want to do is look at the ego lists file.

Reply:
I am trying to implement this for the future in case we do anything else with non-ego Weapons.

Reply:
> This doesn't do anything. There is no mini/city path, what you want to do is look at the ego lists file.

Oh wait, I reread what you said. I made a mini/city path for mini weapons with city properties